README.md

```diff
@@ -1,5 +1,25 @@
 # azure-hanaonazure-cli-extension
 
+# Install
+
+To install this extension just use the CLI extension add command
+
+```
+az extension add --source $LATES_RELEASE
+```
+
+# Usage
+
+To list all hanainstances for the subscription
+```
```

Comment:
nit: In VSCode, you can add an extension to link .md files; this would tell you that there should be a blank line before and after all code blocks.